### PR TITLE
Fixes the issue with showing update available when preview version is installed

### PR DESCRIPTION
**Pull request by vlada-shubina**
If the template package version is preview and it's higher than latest stable, the update notification won't be shown.

### CODE REVIEW

**Comment by vlada-shubina** [comment]
There is another edge case which not handled by this fix: the case when the feed has only preview packages installed. This handling should be added.

**Comment by DavidKarlas** [comment]
I wonder if we should still add `--preview` flag, for example if user wants to check if there is preview available when running `--check-updates`

File: src/Microsoft.TemplateEngine.Edge/Installers/NuGet/NuGetInstaller.cs [hunk 1]
```diff
@@ -86,11 +86,6 @@ public async Task<IReadOnlyList<CheckUpdateResult>> GetLatestVersionAsync(IEnume
                 {
                     if (source is NuGetManagedTemplatesSource nugetSource)
                     {
-                        if (nugetSource.LocalPackage)
-                        {
-                            //updates for locally installed packages are not supported
-                            return Task.FromResult(CheckUpdateResult.CreateSuccessNoUpdate(source));
-                        }
```
[the remark this answers is not in the document]
Comment (vlada-shubina) [reply src/Microsoft.TemplateEngine.Edge/Installers/NuGet/NuGetInstaller.cs:93]: Correct

**Comment by vlada-shubina** [comment]
> I wonder if we should still add `--preview` flag, for example if user wants to check if there is preview available when running `--check-updates`

I added it to discussion items for next UX review

**Pull request merged**